polkadot-protocol/protocol-components/accounts/account-balance-types.md

```diff
@@ -0,0 +1,112 @@
+```
+
+where `free`, `frozen` and `on_hold` are defined above. The `ED` is the the **existential deposit**.
+
+!!! note
+    Wallet providers may display only **spendable**, **locked**, and **reserved** balances.
+
+## Locks
+
+Locks are abstractions over an account's free balance, preventing it from being spent. Multiple locks can overlap on the same balance, rather than being stacked. 
+
+Locks are automatically applied when an account participates in on-chain activities such as staking or voting, but they are not customizable by users.
+
+Locks are included in the account's `frozen` balance, which represents funds that may be free but are non-transferable. This balance is tied up in activities such as [staking](path to staking page), [governance](path to governance page) and [vesting](path to Vested Transfers subsection in Types of Txs)
+
+Locks can overlap in both amount and duration, following these general rules:
+
+1. If multiple locks involve different amounts of tokens, the largest lock determines the total amount of locked tokens.
+2. If multiple locks involve the same amount of tokens, the lock with the longest duration dictates when those tokens can be unlocked.
+
+### Locks Example
+
+Let's take, for example, 80 DOT as a frozen balance. These 80 DOT are currently used in staking and governance as follows:
+
+- 80 DOT Staking (just unbonded) -> lock 28 days
+- 24 DOT OpenGov 1x conviction (referendum just ended, winning side) -> lock 7 days
+- 4 DOT OpenGov 6x conviction (referendum just ended, winning side) -> lock 224 days
+
+![](/images/polkadot-protocol/protocol-components/accounts/locks-example-1.webp)
+
+The 1 DOT ED represents the existential deposit. The total locked amount is 80 DOT, not 108 DOT - Because the largest lock determines the total amount of locked tokens.
+
+However, these 80 DOT will become available for unlock at different times: 
+
+1. First, you must remove the governance lock on 24 DOT after 7 days. 
+
+2. Next, you can unlock the 80 DOT from staking after 28 days. 
+
+3. Finally, after 224 days, you will be able to remove the second governance lock.
+
+![](/images/polkadot-protocol/protocol-components/accounts/locks-example-2.webp)
+
+After 224 days, all 80 DOT (minus existential deposit) will be free and transferrable.
+
+## Edge Cases for Locks
+
+When you use different convictions while having ongoing locks, the longest period and the largest amount are taken into account.
+
+Following the previous example, if you:
+
+1. Undelegate a 1x conviction delegation of 24 DOT, you will incur a 7-day lock on that 24 DOT.
+2. Delegate 4 DOT with a 6x conviction.
+3. Undelegate again before the 7-day lock from the 1x conviction is removed.
+
+You will get a 6x conviction for 24 DOT! 
+
+!!!info
+    Click [here](https://substrate.stackexchange.com/questions/5067/delegating-and-undelegating-during-the-lock-period-extends-it-for-the-initial-am){target=\_blank} for more information about this edge case.
```

Comment:
Using `here` for a link is not a good practice. Replace with a more self-descriptive name. It could be something like Check this [post]() on Stack Exchange or smth like that